template_feed/Microsoft.DotNet.Common.ItemTemplates/content/RazorPage/.template.config/template.json

```diff
@@ -0,0 +1,36 @@
+{
+  "author": "Microsoft",
+  "classifications": ["Web", "ASP.NET"],
+  "name": "Razor Page",
+  "generatorVersions": "[1.0.0.0-*)",
+  "groupIdentity": "Microsoft.AspNetCore.Mvc.RazorPage",
+  "precedence": "100",
+  "identity": "Microsoft.AspNetCore.Mvc.RazorPage",
+  "shortName": "page",
+  "sourceName": "Index",
+  "primaryOutputs": [ { "path": "Index.cshtml"}, { "path": "Index.cs" } ],
+  "defaultName": "Index",
+  "sources": {
+    "modifiers": [
+      {
+        "condition": "(no-pagemodel)",
+        "exclude": [
+          "Index.cs"
+        ]
+      }
+    ]
+  },
+  "symbols": {
+    "namespace": {
```

Comment:
@rynowak - you had asked me about making a parameter required. You can add 
`"isRequired": "true" `
... to the symbol definition to make it required.

Note: I'm tracking down a bug related to isRequired, so you may want to hold off on making that change.